src/helpers/config.js

```diff
@@ -1,3 +1,3 @@
-export const apiUrl = 'https://api.masteraula.com.br';
+export const apiUrl = 'http://localhost:8000';
```

Comment:
```suggestion
export const apiUrl = 'https://api.masteraula.com.br';
```
deveríamos manter a url da api nos merges :)